capstone-ui/src/components/Contact/Contact.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./Contact.css";
+
+function Contact() {
+  return (
+    <div className="contact">
+      <div className="contact-links">
+        <p>
+          <a href="#facebook">
+            <img src="./src/images/facebook.png" height="20"></img>
```

Comment:
It would be good to have a screenshot that covers the whole page in the test plan / description. 
That way we can visualize what these images look like.

Reply:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/57270552/179642605-8e6f0670-430d-4b2e-80aa-42beefde5906.png">
The images are actually not showing up, this is an issue I am yet to resolve